src/EnergyPlus/EcoRoofManager.cc

```diff
@@ -757,6 +540,204 @@ namespace EcoRoofManager {
+                            OutputProcessor::SOVTimeStepType::Zone,
+                            OutputProcessor::SOVStoreType::State,
+                            "Environment");
+        SetupOutputVariable(state,
+                            "Green Roof Vegetation Moisture Transfer Rate",
+                            OutputProcessor::Unit::m_s,
+                            thisEcoRoof->Vfluxf,
+                            OutputProcessor::SOVTimeStepType::Zone,
+                            OutputProcessor::SOVStoreType::State,
+                            "Environment");
+        SetupOutputVariable(state,
+                            "Green Roof Soil Moisture Transfer Rate",
+                            OutputProcessor::Unit::m_s,
+                            thisEcoRoof->Vfluxg,
+                            OutputProcessor::SOVTimeStepType::Zone,
+                            OutputProcessor::SOVStoreType::State,
+                            "Environment");
+        SetupOutputVariable(state,
+                            "Green Roof Vegetation Latent Heat Transfer Rate per Area",
+                            OutputProcessor::Unit::W_m2,
+                            thisEcoRoof->Lf,
+                            OutputProcessor::SOVTimeStepType::Zone,
+                            OutputProcessor::SOVStoreType::State,
+                            "Environment");
+        SetupOutputVariable(state,
+                            "Green Roof Soil Latent Heat Transfer Rate per Area",
+                            OutputProcessor::Unit::W_m2,
+                            thisEcoRoof->Lg,
+                            OutputProcessor::SOVTimeStepType::Zone,
+                            OutputProcessor::SOVStoreType::State,
+                            "Environment");
+
+        SetupOutputVariable(state,
+                            "Green Roof Cumulative Precipitation Depth",
+                            OutputProcessor::Unit::m,
+                            thisEcoRoof->CumPrecip,
+                            OutputProcessor::SOVTimeStepType::Zone,
+                            OutputProcessor::SOVStoreType::State,
+                            "Environment");
+        SetupOutputVariable(state,
+                            "Green Roof Cumulative Irrigation Depth",
+                            OutputProcessor::Unit::m,
+                            thisEcoRoof->CumIrrigation,
+                            OutputProcessor::SOVTimeStepType::Zone,
+                            OutputProcessor::SOVStoreType::State,
+                            "Environment");
+        SetupOutputVariable(state,
+                            "Green Roof Cumulative Runoff Depth",
+                            OutputProcessor::Unit::m,
+                            thisEcoRoof->CumRunoff,
+                            OutputProcessor::SOVTimeStepType::Zone,
+                            OutputProcessor::SOVStoreType::State,
+                            "Environment");
```

Comment:
With the refactor, it's hard to tell exactly what change was made.  However, based on the original issue I assumed it was a change to the setup output variable calls, and it does appear the arguments are different now, and this seems to be good.